examples/publish/publish.go

```diff
@@ -33,7 +33,7 @@ func main() {
 
 		var t = time.Now().Format(time.RFC3339)
```

Comment:
`t := time.Now().Format(time.RFC3339)`. Also drop the `\n` from the `log.Printf`

Reply:
might also want to rename `response` to `resp`